app.js

```diff
@@ -0,0 +1,20 @@
+// Code goes here
+
+(function() {
+
+  var app = angular.module("restaurantViewer", ["ngRoute"]);
+
+  app.config(function($routeProvider){
+        $routeProvider
+            .when("/main", {
+                templateUrl: "main.html",
+                controller: "MainController"
+            })
+            .when("/search/:distance?/:latitude?/:longitude?", {
```

Comment:
This routing is kind of shitty. The question marks mean that the parameters are optional. So this page will take a route like `/search` or `/search/10/20/20` or `/search//20/20`. It's not very explicit what number means what. I'd like to improve this part later.